runtime/compiler/optimizer/SPMDParallelizer.cpp

```diff
@@ -1369,6 +1369,30 @@ bool TR_SPMDKernelParallelizer::reductionLoopExitProcessing(TR::Compilation *com
    return true;
    }
 
+TR::Node * replaceAndAnchorOldNode(TR::Compilation *comp, TR::TreeTop *treeTop, TR::Node *parent, TR::Node *oldNode, TR::Node *newNode, int index)
+   {
+	   treeTop->insertBefore(TR::TreeTop::create(comp, TR::Node::create(TR::treetop, 1, oldNode)));
+	   oldNode->recursivelyDecReferenceCount();
+	   parent->setAndIncChild(index, newNode);
+	   return newNode;
```

Comment:
Let's not return the value passed as a parameter. Better to make that function void.